src/views/homepage.tsx

```diff
@@ -0,0 +1,55 @@
+import Html from '@kitajs/html'
+import { singleton } from 'tsyringe'
+import { Page } from './common.js'
+
+@singleton()
+export default class HomepageTemplates {
+  constructor() {}
+  public homepage = () => {
+    return (
+      <Page title="Veritable - Homepage" heading="Homepage" headerLinks={[{ name: '', url: '/' }]}>
+        <div
+          class="main connections"
+          hx-get="/"
+          hx-trigger="every 10s"
+          hx-select="#search-results"
+          hx-target="#search-results"
+          hx-swap="outerHTML"
+          hx-include="#connection-search-input"
+        ></div>
+        <div class="center-category">
+          <div class="category-container">
+            <div class="category-item" onclick="location.href='#';">
```

Comment:
Should one of these not take us to the connections page? Can you ask Esther please